glide.yaml

```diff
@@ -14,3 +14,4 @@ import:
   version: 378d26f46672a356c46195c28f61bdb4c0a781dd
 - package: github.com/wavefrontHQ/go-metrics-wavefront
 - package: github.com/rcrowley/go-metrics
+- package: github.com/mackerelio/go-osstat/cpu
```

Comment:
Run `make vendor-get pkg=github.com/mackerelio/go-osstat` to automatically insert this entry and download it to the vendor directory.

Reply:
Still failing the CI, why?